lib/util/include/hse_util/data_tree.h

```diff
@@ -3,46 +3,27 @@
  * Copyright (C) 2015-2022 Micron Technology, Inc.  All rights reserved.
  */
 
-#ifndef HSE_PLATFORM_DATA_TREE_H
-#define HSE_PLATFORM_DATA_TREE_H
+#ifndef HSE_UTIL_DATA_TREE_H
+#define HSE_UTIL_DATA_TREE_H
 
 #include <stdbool.h>
 
 #include <cjson/cJSON.h>
 #include <rbtree.h>
 
 #include <hse/error/merr.h>
-
 #include <hse_util/compiler.h>
-#include <hse_util/inttypes.h>
 #include <hse_util/list.h>
 
-typedef enum {
-    DT_FIELD_INVALID,
-    DT_FIELD_ODOMETER_TIMESTAMP,
-    DT_FIELD_ODOMETER,
-    DT_FIELD_LEVEL,
-    DT_FIELD_FLAGS,
-    DT_FIELD_ENABLED,
-    DT_FIELD_CLEAR,
-    DT_FIELD_DATA,
-} dt_field_t;
-
-/* Operations */
-enum { DT_OP_EMIT, DT_OP_COUNT };
-
 /* clang-format on */
 
-/* DT_PATH_SZ restricts the path size of a statically allocated dt_element
- * to avoid wasting too much space.
+/* DT_PATH_MAX restricts the path size of a statically allocated dt_element to
+ * avoid wasting too much space.
```

Comment:
That "to" looks lonely hanging off the end of the line, but if you moved it back down there'd be one less changed line!  ;-)

Reply:
I moved it to reflow the comment to 80 column width which is what I thought we shot for for comments. Plus, I already had to touch the comment anyways.